welcome/oke_about.adoc

```diff
@@ -66,7 +67,7 @@ and {product-title} in the following table:
 | Yes
 
 2+h|Platform Logging
-| Yes
+| 
```

Comment:
See [Table 1. Product comparison for OpenShift Kubernetes Engine and OpenShift Container Platform](https://66035--docspreview.netlify.app/openshift-enterprise/latest/welcome/oke_about#about_oke_similarities_and_differences)

Reply:
What the final name to {product-title}?    OKE or OCP?

Reply:
![image](https://github.com/openshift/openshift-docs/assets/350907/1ce96004-695e-400e-bae3-f62e4bd16f05)

Reply:
Here, `{product-title}` is OpenShift Container Platform.